packages/bls-kms-local/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@sphereon/ssi-sdk-bls-key-manager",
+  "description": "Sphereon KMS implementation that provides BLS crypto",
+  "version": "0.1.0",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "veramo": {
+    "pluginInterfaces": {
+      "IBlsKeyManager": "./src/types/IBlsKeyManager.ts"
+    }
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "generate-plugin-schema": "yarn veramo dev generate-plugin-schema"
+  },
+  "dependencies": {
+    "@mattrglobal/bbs-signatures": "^1.0.0",
+    "@mattrglobal/node-bbs-signatures": "^0.13.0",
+    "@veramo/core": "3.1.2-next.84"
```

Comment:
Do we really need this version, because this is likely going to mess with other plugins

Reply:
Actually Veramo adds partial support to bls keys from version 3.1.2-next.1 on. It would be nice to use the existing implementation.

Reply:
Seems like everything still works with the fixed integration test, so yes we can update